libs/payments/ui/src/lib/client/components/SignInForm/en.ftl

```diff
@@ -7,3 +7,6 @@ next-new-user-subscribe-product-updates-mdnplus = I’d like to receive product
 next-new-user-subscribe-product-updates-mozilla = I’d like to receive product news and updates from { -brand-mozilla }
 next-new-user-subscribe-product-updates-snp = I’d like to receive security and privacy news and updates from { -brand-mozilla }
 next-new-user-subscribe-product-assurance = We only use your email to create your account. We will never sell it to a third party.
+
+checkout-enter-your-email = Enter your email
+checkout-continue = Continue
```

Comment:
Ah, that's my bad, we have these two on L1 and L2 - can we update the IDs in the component instead?

Reply:
Ah sweet thank you!